src/mugs.js

```diff
@@ -475,6 +475,17 @@ define([
         }
     });
 
+    Object.defineProperty(Mug.prototype, "previousSibling", {
+        get: function () {
+            var tree = this.form.tree,
+                node = this.form.tree.getNodeFromMug(this),
+                parentMug = this.parentMug,
+                parentNode = parentMug ? tree.getNodeFromMug(parentMug) : tree.getRootNode(),
+                mugPosition = parentNode.children.indexOf(node);
+            return (mugPosition === 0) ? parentMug : parentNode.children[mugPosition - 1].getValue();
```

Comment:
Also, consider adding a docstring to explain what "previous sibling" means in the context of

- a question whose previous sibling is it's parent
- the first question in the tree
- any other non-obvious details